cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -337,6 +336,11 @@ func (dc *DownstreamController) syncRuleEndpoint() {
 // Start DownstreamController
 func (dc *DownstreamController) Start() error {
 	klog.Info("start downstream controller")
+
+	if err := dc.initLocating(); err != nil {
+		return err
+	}
```

Comment:
duplicate with https://github.com/kubeedge/kubeedge/blob/d2b2e47456547257bd2ca8919040db456ad9999a/cloud/pkg/edgecontroller/controller/downstream.go#L461-L463
remove one of the two?